Author: Michael Hofer <michael.hofer@adfinis-sygroup.ch>

Remove deprecated encoding tag to be compliant with the freedesktop desktop
entry specification and rename referenced icon to be aligned with the app-id.

--- xchat-2.8.8/xchat.desktop.orig	2018-04-25 00:27:24.633151641 +0200
+++ xchat-2.8.8/xchat.desktop	2018-04-25 00:27:54.453151324 +0200
@@ -1,5 +1,4 @@
 [Desktop Entry]
-Encoding=UTF-8
 Name=XChat IRC
 Name[zh_TW]=網路清談
 Comment[de]=IRC-Client
@@ -16,7 +15,7 @@
 Comment[zh_TW]=X-Chat 聊天程式
 Comment=Chat with other people using Internet Relay Chat
 Exec=xchat
-Icon=xchat
+Icon=io.github.Xchat
 Terminal=false
 Type=Application
 Categories=Network;
